scripts/mbedtls_dev/bignum_core.py

```diff
@@ -605,7 +607,99 @@ def generate_function_tests(cls) -> Iterator[test_case.TestCase]:
             cur_op = cls(a, b, n, case_description="replay")
             cur_op.set_limbs(limbs_an4, limbs_b4, limbs_an8, limbs_b8)
             yield cur_op.create_test_case()
-        # Test cases generated randomly
+        # Random test cases can be generated using mpi_modmul_case_generate()
+        # Uses a mixture of primes and odd numbers as N, with four randomly
+        # generated cases for each N.
         for a, b, n, description in cls.random_test_cases:
             cur_op = cls(a, b, n, case_description=description)
             yield cur_op.create_test_case()
+
+
+def mpi_modmul_case_generate() -> None:
```

Comment:
I can't see where this function is called from. What am I missing?

Reply:
This is equivalent to `mpi-modmul-gen.pl` from f334d9622b4c327c56ff081c39fd4b5cb3a92753, I've included this so it can be called if a new set of random test cases are needed for `mpi_core_montmul`.